client/app/dedicated/server/terminate/dedicated-server-terminate.html

```diff
@@ -13,7 +13,10 @@
                 <oui-spinner></oui-spinner>
             </div>
 
-            <div data-ng-if="!ctrl.cancelSubscriptionForm.isSubmiting">
+            <p data-ng-if="server.engagement"
+               data-translate="server_resiliate_contact_support"></p>
```

Comment:
The translation should either be duplicated in the `dedicated/server` translation or moved to a more `common` translation file